restricted_site_access.php

```diff
@@ -451,18 +459,21 @@ public static function restrict_access_check( $wp ) {
 				if ( ! empty( self::$rsa_options['redirect_url'] ) ) {
 					if ( ! empty( self::$rsa_options['redirect_path'] ) ) {
 						$redirect_url_domain = wp_parse_url( self::$rsa_options['redirect_url'], PHP_URL_HOST );
```

Comment:
Looks like `$redirect_url_domain` variable isn't used in the function anymore, this line could be deleted